jscomp/core/lam_compat.ml

```diff
@@ -68,15 +68,15 @@ let cmp_nativeint (cmp : comparison) (a : nativeint) b : bool =
   | Clt -> a < b
   | Cge -> a >= b
 
-let cmp_float (cmp : comparison) (a : float) b : bool =
+let cmp_bigint (cmp : comparison) (a : string) b : bool =
   match cmp with
   | Ceq -> a = b
   | Cneq -> a <> b
   | Cgt -> a > b
```

Comment:
these are not going to work for string right?

Reply:
Oh, I think I missed it, you're correct.

Reply:
https://github.com/rescript-lang/rescript-compiler/pull/6670/commits/acebd8fa804f92afd6a5a21dbbf4439feecefb53